makefile.shade

```diff
@@ -34,7 +34,7 @@ k-standard-goals
 
       var version = Path.GetFileNameWithoutExtension(existingNupkg).Substring(project.Length + 1);
       var nuspecPath = Path.Combine("src", project, project + ".nuspec");
-      var basePath = Path.Combine(outputDir, project, configuration, "net451", "win7-x64");
+      var basePath = Path.Combine(outputDir, project, configuration);
```

Comment:
This doesn't affect any other tool as of this PR. But you'll need to update #124 to not target net451 once this goes in.